### PR TITLE
Bedrock custom models

### PR DESCRIPTION
**Pull request by suyinw**
*Issue #, if available:*

*Description of changes:*
Create the 10_CustomModels directory with contents, update the readme in main and 10_CustomModels directory to include the new contents. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by smith-kyle** [comment]
I noticed you all aren't using a notebook review tool and wanted to invite you to review this pull request with GitNotebooks: https://gitnotebooks.com/aws-samples/amazon-bedrock-workshop/pull/187

It lets you do things like comment on rendered markdown and code cells, so might be an easy win for your PR reviews.

**Comment by suyinw** [comment]
Updates: 
- Added dependency installation of tokenizers==0.12.1 in 00_setup.ipynb; added setup tips with the full list of dependencies for reference
- Added installation of datasets in 00_setup.ipynb
- Removed the installation of fmeval package in 02 notebook as it's done in 00
- Removed the comment to select the modelId from the list, instead provided the reference model link from bedrock developer guide in 02 notebook
- Additionally, added the code block in 02 nb to visualize the training/validation loss for fine-tuning.

**Comment by suyinw** [comment]
Dependencies specifications and additional notes

- updates main readme with estimated time for running the whole directory
- updates to CustomModels readme to remove the requirement to run the bedrock_boto3_setup.ipynb as it's not necessary for this module
- updated the packages with specific versions in 00_setup.ipynb notebook to ensure it runs stably across environments
- moved the fmeval package installation from 00_setup to the 02_fine-tune_and_evaluate_llama2_bedrock_summarization.ipynb notebook as it's specific to this one notebook
 - added the code for s3 create_bucket() with locationconstraint to make sure it works across different regions
 - added more notes/warnings in 02_fine-tune_and_evaluate_llama2_bedrock_summarization.ipynb to help with successful runs
 - added notebook instance specifications in 04_cleanup.ipynb

**Review by ssinghgai — APPROVED** [review]
Please add instructions in notebook 02 to to Cleanup notebook for complete CleanUp. OR add all cleanup instructions in 02 notebook.

**Comment by ssinghgai** [comment]
Approved!
